### DISCUSSION TITLE
Is it possible to use a combination of nextjs and vuejs?

### DISCUSSION THREAD

**Discussion by khoand081** [vercel/next.js#15364] [Help] [answered]
I was disoriented when I wanted to use react-google-charts in the project vuejs, I tried and I knew that without the reactjs environment react-google-charts wouldn't work. Thanks!

**Comment by jamesmosier** [comment, marked as answer]
Hi @khoand081. You're correct, it is not possible to mix the 2 frameworks with Next.js. Next.js uses React for rendering and it built into the framework.
